crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -88,6 +90,8 @@ pub struct AuthorityPerEpochStore {
     /// A write-ahead/recovery log used to ensure we finish fully processing certs after errors or
     /// crashes.
     wal: Arc<DBWriteAheadLog<TrustedCertificate, (InnerTemporaryStore, SignedTransactionEffects)>>,
+    /// Metric for sending tallying rule scores for all validators of this epoch.
+    tallying_rule_score_metric: IntGaugeVec,
```

Comment:
i think it would be better to move this metric to AuthorityMetrics so that we don't need to pass in a Registry to the reconfigure method - but maybe that's trickier than it seems to me.

Reply:
We need to be able to access the metric in the per epoch store. Are you suggesting we maybe put an `Arc<AuthorityMetrics>` in the epoch store?